packages/rollup-dev-tools/src/tools/transpiler/transpiler.ts

```diff
@@ -677,8 +686,24 @@ export class TranspilerImpl implements Transpiler {
       transpiledBytecode,
       jumpdestIndexesBefore
     )
+    // TODO make sure accountForJumps STOPs after, should do
     errors.push(...(res.errors || []))
-    transpiledBytecode = res.bytecode
+    const bytecodeWithExistingJumpsFixed = res.bytecode
```

Comment:
```suggestion
    const bytecodeWithTranspiledJumpdestLocationsPopulated = res.bytecode
```
Might not be the best description, but the suggestion mainly entails replacing `fixed` with a more descriptive word.